i18n/it.toml

```diff
@@ -221,13 +221,21 @@ restartButton = "Ricomincia"
 title = "Errore di avvio"
 hint = "Nota: potrebbe anche trattarsi di un dispositivo difettoso (inverter, contatore, ...). Controllare le connessioni di rete."
 
+[session]
+title = "Sessione di ricarica"
+meterstart = "Conteggio iniziale"
+meterstop = "Conteggio finale"
+odometer = "Chilometraggio"
```

Comment:
Do you speak all these languages or are these Google Translates? We generally only specify translations for languages we speak (de/en). Community translations are done via Weblate https://hosted.weblate.org/projects/evcc/evcc/
There its easy to see untranslated strings and native speakers can translate from English to their own language.

Reply:
This does not mean that you should roll them back. Just an info for your next PR. Must have been a lot of effort to add all these translations yourself. 🌟

Reply:
I wanted to do it very neatly 😄 I have no concerns deleting the translation of languages that aren't familiar to me. Maybe this would be the less complete but more clean way. What do you think?

Reply:
Yes please